images/linux/scripts/installers/7-zip.sh

```diff
@@ -4,6 +4,8 @@
 ##  Desc:  Installs 7-zip
 ################################################################################
 
+set -e
```

Comment:
Why can't we set it in shebang on the first line?

```
#!/bin/bash -e
```

Reply:
Agree, it looks better. The same for -o

Reply:
I believe the packer runs the script as `/bin/bash script.sh` instead of `./script.sh` so the tag is ignored

Reply:
no, it's not. I've checked on macos — `#!/bin/bash -e `works!